pages/common/git-local-commits.md

```diff
@@ -0,0 +1,9 @@
+# git local-commits
+
+> Show local commits which haven't been pushed to origin. Any additional arguments will be passed directly to `git log`.
```

Comment:
I think that sounds slightly more fluent, and it's what's used in the example.
```suggestion
> Show local commits that haven't been pushed to origin. Any additional arguments will be passed directly to `git log`.
```

Reply:
Doesn't matter to me however I have no objections. Committed